docs/manual/figures/value-subtyping.svg

```diff
@@ -1,791 +1,183 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
It looks like the text on the top row (the one with all the annotations next to each other) is a little too compressed on `ArrayLen` and `ArrayLenRange`. Are the `@` symbols actually inside the box entirely? It would be great if we could make the boxes just a little bigger.